src/ophys_etl/transforms/roi_transforms.py

```diff
@@ -0,0 +1,133 @@
+from typing import List, Optional, Tuple
+
+import numpy as np
+from scipy.sparse import coo_matrix
+
+
+def suite2p_rois_to_coo(suite2p_stats: np.ndarray,
```

Comment:
We've been repeating these transforms a lot (binarization, coo format, etc.). Since this repo is the most general we should try to reuse these in the future rather than doing them over again.